### DISCUSSION TITLE
Comunicación confusa

### DISCUSSION THREAD

**Discussion by jmlimas** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#146] [General]
Soy  yo  o  siento  que a  comunicación es  muy confusa,  usar  github  como  medio  de  comunicación   y  los  en vivos  siento que  es  todo esto  muy  confuso o puede ser  mi  incapacidad  de  asimilar

**Comment by Ingrzs** [comment]
Pues mira aquí en github es para que dejen las tareas en tipo codigo etc,y aqui mismo subas tus tareas, también existe un apartado de discusiones para dudas .

Team solo es para live y que ellos pongan los formularios y agendar que tareas dejeran.

Si el comander comenta aqui es porque es para ver como haras las tareas el sube ejemplos etc.

Si el comander comenta en team solo seran avisos

**Comment by AngelCruzO** [comment]
Si puede llegar a ser confuso porque son diferentes canales de comunicación, pero despues entiendes el porque se manejo asi. Los lives se manejan en teams porque pertenece a Microsfot y es una buena plataforma, Github en este caso solo funciona como medio de comunicación para discusiones por como se estructuran, ya que si es mas facil encontrar cada duda, tambien sirve para ver la documentación de cada mision y modulos que se van viendo, es mas facil verlo aqui en github que con un monton de documentos en teams.

Ahora tambien se vuelve confuso, ya que las dudas se repiten mucho, no tenemos las costrumbre de leer antes las dudas que ya existen para ver si ahi solucionan la duda. Pero si al principio si compartia tu estado de confusion

**Comment by Burstniart** [comment]
En sí Github debería mantenerse centrado a temas de trabajo y código, pero como el teams se está usando para avisar de las actividades los discussions se están usando como método de comunicación entre los explorers.
Mientras mantengan orden en los temas que abran es bastante fácil de navegar.
Y los lives son bastante claros, dan la info necesaria y si no hablan de algún tema es porque aún no se llega a ello.

**Comment by JuanSalvador73** [comment]
Creo que a la mayoría nos ha pasado eso, sobretodo si no tenemos experiencia usando Github. Como han dicho arriba, Git es para trabajos y consultas y StrealLaunchX para los tutoriales o los "live". Antes tenía los 2 abiertos en mi labtop, ya no.

Lo que hago ya, es que como ya trabajo directamente en Git, solo tengo abierto este, y si me voy a un "live" - grabado o en vivo - pues te lleva al Streaming directamente. En cuanto termino de verlo lo cierro y regreso al Git. 

Aunque a estas alturas ya debe uno estar trabajando en los softwares para tus tareas, y usar el git solo para trabajo y consulta y ocasionalmente el Streaming LaunchX. Te recomiendo hacerlo así, para evitarte confusiones.

Saludos.
